typescript/web/src/pages/auth/signin.tsx

```diff
@@ -22,7 +22,9 @@ const useRedirectIfAuthenticated = (redirectUrl?: string) => {
 };
 
 const MustLogInMessage = () => (
-  <Heading as="h3">Please sign-in to continue</Heading>
+  <Heading as="h3" textAlign="center" fontSize="3xl">
+    Please sign-in to continue
+  </Heading>
```

Comment:
Thank you for this fix ❤️  It's sad that I didn't moved the whole sign-in page body in a component when I've built the page. We would have it covered with Chromatic if it was the case...

_You can self-resolve this comment once read_

Reply:
I'm not even sure that this message should be here, as it's not in the Figma mockups. I've pushed this change to re-trigger the whole CI as it was failing for whatever reason

Reply:
For the location, it feel non optimal I agree. As it's not the goal of this PR we can keep it as-is and we'll eventually open an issue during today's dsm if needed 😉